cram/cram_encode.c

```diff
@@ -1825,8 +1843,11 @@ int cram_encode_container(cram_fd *fd, cram_container *c) {
             // This starts as 'N' and is amended on-the-fly as we go
             // based on MD:Z tags.
             if ((c->ref_id = bam_ref(b)) >= 0) {
-                c->ref_free = 1;
                 c->ref = NULL;
+                // c->ref_ref is boolean; whether to free c->ref.  In this
```

Comment:
I'm confused by this comment, as there is no `ref_ref` :smile: Should be `c->ref_free`?

Reply:
Doh!  Yes.  Not a helpful comment afterall. :)

Thanks